src/lib/components/preset.svelte

```diff
@@ -100,6 +101,7 @@
 			updateEditor: true,
 			resetPanZoom: true
 		});
+		void logEvent('loadSampleDiagram', { diagramType });
```

Comment:
Do you think it would be better to have named properties for the options of the event like:
```json
{
  diagramType: 'flowchart'
}
```
Instead of
```json
'flowchart'
```
This way you can easily add more properties after

Reply:
That is exactly how the data is sent. 

```ts
{ diagramType } 
```

Is the ES6 shorthand notation for 

```ts
{
  diagramType: diagramType
}
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer

Reply:
Great!